src/Emulator.cpp

```diff
@@ -1,15 +1,13 @@
 #include "PCH.hpp"
 #include "Emulator.hpp"
 
+const Uint32 TimePerFrame = static_cast<Uint32>((1.0 / 60.0) * 1000.0);
+
 Emulator::Emulator() :
     m_isRunning(true),
     m_windowTitle("Gameboy"),
     m_windowWidth(160),
-    m_windowHeight(144),
-    m_window(nullptr),
```

Comment:
I made m_window, m_renderer into unique_ptr too. And you don't need to initialize them, as they are null to start for us.